recyclerview-fastscroll/src/main/java/com/simplecityapps/recyclerview_fastscroll/views/FastScrollRecyclerView.java

```diff
@@ -356,6 +357,10 @@ public void onUpdateScrollbar() {
     /**
      * Returns the current scroll state of the apps rows.
      */
+    private int lastRowIndex;
+    private int lastTopOffset;
+    private int lastRowHeight;
```

Comment:
These class level variables are used here to remember the last states of where the scroller bar is when the item is not a __header__ item (header item is treated differently compared to other items, it has a pan of 3 and height is much shorter too).